lib/component-data/create.test.js

```diff
@@ -94,18 +93,6 @@ describe('component creator', () => {
     });
   });
 
-  it('fetches base component styles for new components', () => {
```

Comment:
because we're just adding link tags, there's not much to mock without mucking with the "global" store, so I'm not sure if this can be unit tested effectively